lib/linked_in_export.rb

```diff
@@ -0,0 +1,56 @@
+require 'net/http'
+require 'net/https'
+require 'nokogiri'
+require 'uri'
+
+module LinkedInExport
+
+  def linked_in_oauth(service, user, session)
+    Rails.logger.debug("### linked_in_oauth - begin for user = #{user.inspect}.")
+    linkedin_connection = LinkedIn::Connection.new
+    request_token = linkedin_connection.request_token(bz_linked_in_export_oauth_success_url)
+    session[:oauth_linked_in_request_token_token] = request_token.token
+    session[:oauth_linked_in_request_token_secret] = request_token.secret
+    OauthRequest.create(
+      :service => 'linked_in',
+      :token => request_token.token,
+      :secret => request_token.secret,
+      :return_url => bz_linked_in_export_oauth_success_url,
+      :user => user,
+      :original_host_with_port => request.host_with_port
+    )
+    Rails.logger.debug("### Calling to oauth against LinkedIn API for user = #{user.inspect}.  Target URL: #{request_token.authorize_url}")
+    response = request_token.request(:get,  request_token.authorize_url, request_token)
+    Rails.logger.debug("### Response = #{response.inspect}, response.body = #{response.body}")
+
+    #http,request = delicious_generate_request('https://api.del.icio.us/v1/posts/update', 'GET', service.service_user_name, service.decrypted_password)
```

Comment:
I was just talking about the delicious sample code, but if we're not going to use any of the file then I guess we should delete it.  I thought the "Export LinkedIn Data" button was still calling it and this is where we would put the code to actually do the export.  Is that wrong though?